content/02.introduction.md

```diff
@@ -165,6 +165,18 @@ This excessive clotting in lung capillaries has been suggested to related to a d
 Individual outcomes have varied significantly, and reported hospitalization rates have been wide-ranging based on age and location, with various underlying health conditions being disproportionately reported [@doi:10.15585/mmwr.mm6915e3].
 Outcomes in patients that are hospitalized have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].
 
+The presentation of COVID-19 infection can vary greatly among pediatric patients, and in some cases they might manifest in distinct ways from COVID-19 in adults. 
+[@doi:10.1001/jamapediatrics.2020.1467] reviewed studies on children infected with COVID-19 during the early months of COVID-19 in China.
```

Comment:
```suggestion
[@doi:10.1001/jamapediatrics.2020.1467] reviewed 17 studies on children infected with COVID-19 during the early months of the COVID-19 epidemic in China and one study from Singapore.
```
I'm not sure if we can say they were in China since one was in Singapore -- but 17/18 were in China, so I don't know what the best way to mention the single Singapore study would be!